tools/config.js

```diff
@@ -63,7 +63,9 @@ Promise.resolve(userToken || cachedToken)
         }
       ]);
 
-      const { projectId: { id } } = response;
+      const {
+        projectId: { id }
+      } = response;
```

Comment:
ditto...  Can you make sure you're running the right prettier? (run yarn from the repo root)

Reply:
Done